tests/transformers/test_transformers_model_export.py

```diff
@@ -1644,16 +1644,15 @@ def test_zero_shot_classification_pipeline(zero_shot_pipeline, model_path, data)
 @pytest.mark.parametrize(
     ("query", "result"),
     [
-        ({"query": "What should we order more of?"}, ["apples"]),
+        ({"query": "What should we order more of?"}, ["Apples"]),
         (
             {
                 "query": [
                     "What is our highest sales?",
                     "What should we order more of?",
-                    "Which fruit starts with 'W'?",
```

Comment:
The tiny model is incapable of reasoning of this nature. We don't need 3 queries anyway; only 2 to validate the multi-record inference capability.